components/04-views/login/_login.scss

```diff
@@ -1,170 +1,123 @@
-$login__title-text-transform                : uppercase !default;
-
-$login__validation-tips-color               : $red !default;
-$login__validation-tips-margin              : 0 0 $spacer !default;
+$login__validation-tips-color           : $red !default;
+$login__validation-tips-margin          : 0 0 $spacer !default;
 
 .login {
     padding-top: $login__padding-top;
     margin-bottom: $login__margin-bottom;
-    &__title {
-        text-transform: $login__title-text-transform;
-        margin-bottom: $login__title-margin-bottom;
-    }
-    &__info {
-        margin: $login__info-margin;
-        text-align: center;
+    &__form-column {
+        flex-basis: 100%;
+        max-width: 100%;
         @include mq($screen-m) {
-            margin: $login__info-margin\@medium;
-            text-align: left;
+            @include grid-column-width(1, 2);
+            padding-right: $login__form-column-padding\@medium;
         }
     }
-    &__content {
+
+    &__link-column {
+        flex-basis: 100%;
+        max-width: 100%;
         @include mq($screen-m) {
-            display: flex;
-            justify-content: space-between;
-            align-items: flex-start;
+            @include grid-column-width(1, 2);
+            padding-left: $login__link-column-padding\@medium;
         }
     }
-    &__fieldset {
-        @include mq ($screen-m) {
-            max-width: 488px;
-        }
-    }
-    &__tabs {
-        margin: $login__tabs-margin;
-        width: $login__tabs-width;
-        @include mq($screen-m) {
-            width: $login__tabs-width\@medium;
-        }
+
+    &__info {
+        margin: $login__info-margin;
     }
-    &__tab-container {
-        margin: $login__tabs-content-margin;
+
+    &__form {
+        margin: $login__form-margin;
+        padding: $login__form-padding;
+        border-bottom: $login__form-border;
         @include mq($screen-m) {
-            max-width: $login__tabs-content-max-width\@medium;
-        }
-        @include mq($screen-l) {
-            max-width: $login__tabs-content-max-width\@large;
-        }
-    }
-    &__tab-content {
-        padding: $login__tab-content-padding;
-        &--active {
-            opacity: 1;
-            z-index: 1;
-            display: block;
+            margin: $login__form-margin\@medium;
+            padding: $login__form-padding\@medium;
+            border-bottom: $login__form-border\@medium;
         }
     }
-    &__sidebar {
-        margin: $login__sidebar-margin;
+
+    &__fieldset {
         @include mq($screen-m) {
-            width: $login__sidebar-width\@medium;
+            max-width: $login__fieldset-max-width\@medium;
         }
     }
+
     &__actions {
         display: flex;
-        flex-wrap: nowrap;
+        flex-wrap: wrap;
         justify-content: flex-start;
         align-items: center;
         margin: $login__actions-margin;
     }
+
     &__button {
+        width: 100%;
+        flex: 0 0 100%;
```

Comment:
Do we need both `flex-basis` and `width` here? I think you can remove `width`, `flex` is enought

Reply:
I have two cases and for one is needed.

Reply:
Ok